server/src/http/instance_data.rs

```diff
@@ -115,7 +115,7 @@ pub async fn loop_send_instance_data(
         match instance_data_sender.as_ref() {
             InstanceDataSending::SendNothing => {
                 debug!("No instance data sender found. Doing nothing.");
-                return;
+                continue;
```

Comment:
Just a suggestion but can we put this in another PR so we can get the last patch version in the minor range working correctly

Reply:
yes. will do